### PR TITLE
[AArch64] support ACL fast math (BF16/F32 mixed-precision) for fp32 conv

### PR DESCRIPTION
**Pull request by wyzero**

### CODE REVIEW

File: tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc [hunk 1]
```diff
@@ -176,6 +176,20 @@ struct MkldnnConvState : public Context::Resource {
 };
 
 #if defined(TAO_AARCH64)
+
+// Returns true is ACL AMP is enabled (`DISC_CPU_ACL_USE_AMP`)
+bool useAclAMP() {
+  static bool flag = []() {
+    const char* env = getenv("DISC_CPU_ACL_USE_AMP");
```
Comment (fortianyou) [inline tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc:183]: Nit: tensorflow::ReadBoolFromEnvVar

Comment (wyzero) [reply tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc:183]: We do not want to introduce the TF dependency in RAL whenever possible.

Comment (fortianyou) [reply tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc:183]: How about implement something like the function utility in RAL?

Comment (wyzero) [reply tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc:183]: OK, I personally prefer to do this in a separate PR, including changing other places of using env flags. I'll send the PR later.

**Review by fortianyou — APPROVED** [review]
LGTM

File: tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc (hunk 1 above)
Comment (fortianyou) [reply tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc:183]: Thanks!

**Pull request merged**